.github/workflows/build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Install packages and dependencies
         run: |
           python -m pip install --upgrade pip wheel
-          pip install -e .
+          pip install -e .[test]
```

Comment:
The udf tests require `pandas`, I can change it but it felt like a natural thing to use with the feature. Since the file imports and uses pandas, the dep is necessary to both collect and run the test. 

Generally, the `test` extra is all deps required to test the project. Are we not following that convention?